### PR TITLE
S3 fan fix

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: src/board/system76/common/power.c [hunk 1]
```diff
@@ -317,8 +321,8 @@ static bool power_peci_limit(bool ac) {
 void power_set_limit(void) {
     static bool last_power_limit_ac = true;
     // We don't use power_state because the latency needs to be low
-#if EC_ESPI
-    if (!in_s0ix) {
+#if USE_S0IX
+    if (gpio_get(&SLP_S0_N)) {
```
Comment (mkopec) [inline src/board/system76/common/power.c:325]: I'd leave this line as it was before, so:

```suggestion
    if (!in_s0ix) {
```

this fixes the LED blinking irregularly while Windows is sleeping by smoothing out the GPIO inputs.

File: src/board/system76/common/power.c [hunk 2]
```diff
@@ -570,8 +574,8 @@ void power_event(void) {
     static uint32_t last_time = 0;
     uint32_t time = time_get();
     if (power_state == POWER_STATE_S0) {
-#if EC_ESPI
-        if (in_s0ix) {
+#if USE_S0IX
+        if (!gpio_get(&SLP_S0_N)) {
```
Comment (mkopec) [inline src/board/system76/common/power.c:578]: Same as above, so:

```suggestion
        if (in_s0ix) {
```

File: src/board/system76/darp8/board.mk [hunk 3]
```diff
@@ -7,6 +7,9 @@ CFLAGS+=-DEC_ESPI=1
 # Apply PMC hack for S0ix
 CFLAGS+=-DPMC_S0IX_HACK=1
 
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
+
```
Comment (mkopec) [inline src/board/system76/darp8/board.mk:12]: We're not using S0ix on these devices anymore, please remove

File: src/board/system76/galp6/board.mk [hunk 4]
```diff
@@ -5,8 +5,8 @@ EC=it5570e
 # Enable eSPI
 CFLAGS+=-DEC_ESPI=1
 
-# Apply PMC hack for S0ix
-CFLAGS+=-DPMC_S0IX_HACK=1
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
```
Comment (mkopec) [inline src/board/system76/galp6/board.mk:9]: As above, this device will not be using S0ix anymore.

File: src/board/system76/galp5/board.mk [hunk 5]
```diff
@@ -5,6 +5,9 @@ EC=it5570e
 # Enable eSPI
 CFLAGS+=-DEC_ESPI=1
 
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
+
```
Comment (mkopec) [inline src/board/system76/galp5/board.mk:10]: As above, this device will not be using S0ix anymore.

File: src/board/system76/galp6/board.mk (hunk 4 above)
Comment (Stojek139808) [reply src/board/system76/galp6/board.mk:9]: Done.

File: src/board/system76/galp5/board.mk (hunk 5 above)
Comment (Stojek139808) [reply src/board/system76/galp5/board.mk:10]: Done.

File: src/board/system76/darp8/board.mk (hunk 3 above)
Comment (Stojek139808) [reply src/board/system76/darp8/board.mk:12]: Done.

File: src/board/system76/common/power.c (hunk 1 above)
Comment (Stojek139808) [reply src/board/system76/common/power.c]: Reverted the change, there was a conflict when cherry-picking and I've chosen the incoming change.

File: src/board/system76/galp6/board.mk (hunk 4 above)
Comment (Stojek139808) [reply src/board/system76/galp6/board.mk:9]: There was an error when pushing which i didn't notice, should be fine now.

**Review by mkopec — APPROVED** [review]

**Pull request merged**